### ISSUE TITLE
PostgreSQL and dtrace.

### ISSUE THREAD

**Issue by dijit** [PostgresApp/PostgresApp#340]
Hello,

I've been using postgresql on FreeBSD and Solaris for some time, and in that time I have learned a few tricks with it's [inbuilt dtrace](https://wiki.postgresql.org/wiki/DTrace) probes.

When compiling the official postgresql.app is it possible to use the configure flags required to instantiate dtrace probes?

the flag is `--enable-dtrace` as documented [here](https://www.postgresql.org/docs/current/static/install-procedure.html)

thanks!

**Comment by jakob** [comment]
This sounds like a reasonable request. Is there a downside to enabling dtrace? I'm wondering why there is a configure flag, and why it isn't enabled by default.

**Comment by dijit** [comment]
There's no dtrace on Linux platforms so if it was default they would fail.

**Comment by felixge** [comment]
I agree, it'd be great to have Postgres.app ship with `--enable-dtrace`.
